addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -385,25 +385,26 @@ def test_message_notify(self):
             'customer_id': self.partner_1.id,
             'name': 'Not A Thread',
         })
-        with self.assertPostNotifications([{
-                'content': 'Hello Paulo',
-                'email_values': {
-                    'reply_to': self.company_admin.catchall_formatted,
-                },
-                'message_type': 'user_notification',
-                'notif': [{
-                    'check_send': True,
-                    'is_read': True,
-                    'partner': self.partner_2,
-                    'status': 'sent',
-                    'type': 'email',
-                }],
-                'subtype': 'mail.mt_note',
-            }]):
-            _message = self.env['mail.thread'].message_notify(
-                body='<p>Hello Paulo</p>',
-                model=test_record._name,
-                res_id=test_record.id,
-                subject='Test Notify',
-                partner_ids=self.partner_2.ids
-            )
+        for subject in ["Test Notify", False]:
```

Comment:
Would add in docstring of test method why we test with a fixed and void subject :)